make quicklint doesn't work for me:

```
ezyang-mbp:pytorch-tmp ezyang$ make quicklint -j10
./.github/scripts/generate_linux_ci_workflows.py
  File "  File "  File "  File "tools/actions_local_runner.pytools/actions_local_runner.pytools/actions_local_runner.pytools/actions_local_runner.py", line ", line ", line ", line 32323232



                def should_color() -> bool:
def should_color() -> bool:
def should_color() -> bool:
def should_color() -> bool:
                                                                                 ^
     SyntaxError  ^
 :   SyntaxErrorinvalid syntax^
:  
invalid syntax^
SyntaxError
: SyntaxErrorinvalid syntax: 
invalid syntax
make: *** [cmakelint] Error 1
make: *** Waiting for unfinished jobs....
make: *** [quick_checks] Error 1
make: *** [flake8] Error 1
make: *** [mypy] Error 1
Traceback (most recent call last):
  File "/Users/ezyang/Dev/pytorch-tmp/./.github/scripts/generate_linux_ci_workflows.py", line 5, in <module>
    import jinja2
ModuleNotFoundError: No module named 'jinja2'
make: *** [generate-gha-workflows] Error 1
```

--- a/test/test_sparse_csr.py
+++ b/test/test_sparse_csr.py
@@ -22,8 +22,8 @@ class TestSparseCSR(TestCase):
         col_indices = [0, 1, 0, 1]
         values = [1, 2, 3, 4]
         sparse = torch._sparse_csr_tensor(torch.tensor(crow_indices, dtype=torch.int64),
-                                         torch.tensor(col_indices, dtype=torch.int64),
-                                         torch.tensor(values), dtype=dtype, device=device)
+                                          torch.tensor(col_indices, dtype=torch.int64),
+                                          torch.tensor(values), dtype=dtype, device=device)
         self.assertEqual(torch.tensor(crow_indices, dtype=torch.int64), sparse.crow_indices())
         self.assertEqual((len(crow_indices) - 1, max(col_indices) + 1), sparse.shape)
         self.assertEqual(dtype, sparse.dtype)
@@ -38,11 +38,11 @@ class TestSparseCSR(TestCase):
         values = [1, 2, 3, 4]
         for index_dtype in [torch.int32, torch.int64]:
             sparse = torch._sparse_csr_tensor(torch.tensor(crow_indices, dtype=index_dtype),
-                                             torch.tensor(col_indices, dtype=index_dtype),
-                                             torch.tensor(values),
-                                             size=(2, 10),
-                                             dtype=dtype,
-                                             device=device)
+                                              torch.tensor(col_indices, dtype=index_dtype),
+                                              torch.tensor(values),
+                                              size=(2, 10),
+                                              dtype=dtype,
+                                              device=device)
             self.assertEqual((2, 10), sparse.shape)
             self.assertEqual(torch.tensor(crow_indices, dtype=index_dtype), sparse.crow_indices())
             self.assertEqual(torch.tensor(col_indices, dtype=index_dtype), sparse.col_indices())
@@ -56,26 +56,26 @@ class TestSparseCSR(TestCase):
         values = [1, 2, 3, 4]
         size = (2, 10)
         torch._sparse_csr_tensor(torch.tensor(crow_indices), torch.tensor(col_indices), torch.tensor(values), size,
-                                dtype=dtype, device=device)
+                                 dtype=dtype, device=device)
 
         with self.assertRaisesRegex(RuntimeError,
                                     r"crow_indices\.numel\(\) must be size\(0\) \+ 1, but got: 3"):
             torch._sparse_csr_tensor(torch.tensor(crow_indices), torch.tensor(col_indices), torch.tensor(values), (1, 1),
-                                    dtype=dtype, device=device)
+                                     dtype=dtype, device=device)
 
         with self.assertRaisesRegex(RuntimeError, "0th value of crow_indices must be 0"):
             torch._sparse_csr_tensor(torch.tensor([-1, -1, -1]), torch.tensor(col_indices), torch.tensor(values), size,
-                                    dtype=dtype, device=device)
+                                     dtype=dtype, device=device)
 
         with self.assertRaisesRegex(RuntimeError, "last value of crow_indices should be less than length of col_indices."):
             torch._sparse_csr_tensor(torch.tensor(crow_indices), torch.tensor([0, 0, 0]), torch.tensor(values), size,
-                                    dtype=dtype, device=device)
+                                     dtype=dtype, device=device)
 
         with self.assertRaisesRegex(RuntimeError,
                                     r"col_indices and values must have equal sizes, " +
                                     r"but got col_indices\.size\(0\): 4, values\.size\(0\): 5"):
             torch._sparse_csr_tensor(torch.tensor(crow_indices), torch.tensor(col_indices), torch.tensor([0, 0, 0, 0, 0]),
-                                    size, dtype=dtype, device=device)
+                                     size, dtype=dtype, device=device)
 
     @onlyCPU
     def test_sparse_csr_print(self, device):
@@ -147,7 +147,7 @@ class TestSparseCSR(TestCase):
         col_indices = torch.tensor([0, 1, 2, 0, 1])
         values = torch.tensor([1, 2, 1, 3, 4], dtype=dtype)
         csr = torch._sparse_csr_tensor(crow_indices, col_indices,
-                                      values, dtype=dtype, device=device)
+                                       values, dtype=dtype, device=device)
         dense = torch.tensor([[1, 2, 1], [3, 4, 0]], dtype=dtype, device=device)
         self.assertEqual(csr.to_dense(), dense)
 
@@ -187,8 +187,8 @@ class TestSparseCSR(TestCase):
         if torch.has_mkl:
             for index_dtype in [torch.int32, torch.int64]:
                 sp = torch._sparse_csr_tensor(torch.tensor([0, 2, 4]),
-                                             torch.tensor([0, 1, 0, 1]),
-                                             torch.tensor([1, 2, 3, 4], dtype=dtype, device=device))
+                                              torch.tensor([0, 1, 0, 1]),
+                                              torch.tensor([1, 2, 3, 4], dtype=dtype, device=device))
                 vec = torch.randn((2, 1), dtype=dtype, device=device)
                 with warnings.catch_warnings(record=True) as w:
                     sp.matmul(vec)

--- a/tools/pyi/gen_pyi.py
+++ b/tools/pyi/gen_pyi.py
@@ -292,9 +292,9 @@ def gen_pyi(native_yaml_path: str, deprecated_yaml_path: str, fm: FileManager) -
                               ' size: Optional[_size]=None, *, dtype: Optional[_dtype]=None,'
                               ' device: Union[_device, str, None]=None, requires_grad:_bool=False) -> Tensor: ...'],
         '_sparse_csr_tensor' : ['def _sparse_csr_tensor(crow_indices: Tensor, col_indices: Tensor,'
-                               ' values: Tensor, size: Optional[_size]=None,'
-                               ' *, dtype: Optional[_dtype]=None,'
-                               ' device: Union[_device, str, None]=None, requires_grad:_bool=False) -> Tensor: ...'],
+                                ' values: Tensor, size: Optional[_size]=None,'
+                                ' *, dtype: Optional[_dtype]=None,'
+                                ' device: Union[_device, str, None]=None, requires_grad:_bool=False) -> Tensor: ...'],
         '_sparse_coo_tensor_unsafe': ['def _sparse_coo_tensor_unsafe(indices: Tensor, values: Tensor, size: List[int],'
                                       ' dtype: Optional[_dtype] = None, device: Optional[_device] = None,'
                                       ' requires_grad: bool = False) -> Tensor: ...'],

--- a/torch/_tensor.py
+++ b/torch/_tensor.py
@@ -947,8 +947,8 @@ class Tensor(torch._C._TensorBase):
                 ro.append(i)
 
             return torch._sparse_csr_tensor(torch.tensor(ro, dtype=row_indices.dtype),
-                                           coalesced_self.indices()[1], coalesced_self.values(),
-                                           size=coalesced_self.shape, dtype=coalesced_self.dtype)
+                                            coalesced_self.indices()[1], coalesced_self.values(),
+                                            size=coalesced_self.shape, dtype=coalesced_self.dtype)
         elif self.is_sparse_csr:
             return self
         else:

--- a/torch/testing/_internal/common_utils.py
+++ b/torch/testing/_internal/common_utils.py
@@ -1075,8 +1075,8 @@ class TestCase(expecttest.TestCase):
 
         values, crow_indices, col_indices = random_sparse_csr(size[0], size[1], nnz)
         return torch._sparse_csr_tensor(crow_indices,
-                                       col_indices,
-                                       values, size=size, dtype=dtype, device=device)
+                                        col_indices,
+                                        values, size=size, dtype=dtype, device=device)
 
     def genSparseTensor(self, size, sparse_dim, nnz, is_uncoalesced, device, dtype):
         # Assert not given impossible combination, where the sparse dims have

